src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -731,6 +733,24 @@ public XmlDocCommentTextFormatDefinition()
         }
         #endregion
 
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.TestCode)]
+        [Name(ClassificationTypeNames.TestCode)]
+        [Order(After = ClassificationTypeNames.StringLiteral)]
+        [Order(After = ClassificationTypeNames.VerbatimStringLiteral)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class TestCodeFormatDefinition : ClassificationFormatDefinition
+        {
+            [ImportingConstructor]
+            [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+            public TestCodeFormatDefinition()
+            {
+                this.DisplayName = "test code";
+                this.BackgroundColor = Color.FromRgb(60, 60, 60);
```

Comment:
WIP: would need actual name and xml definition for the colors.

Reply:
Does the name need to be "CSharp Test Code"? Or does it not work like that?

Reply:
This is the text that will show up in "fonts and colors" so it would be whatever we felt was appropriate there. This was more to just get something up and running :-)